File: gmall-realtime/src/main/scala/com/atguigu/utils/PropertiesUtil.scala
package com.atguigu.utils

import java.io.InputStreamReader
import java.util.Properties

/**
 * @author Chijago
 * @create 2020-02-20 19:56
 */
object PropertiesUtil {

  def load(propertiesName: String): Properties = {
    val prop = new Properties()
    prop.load(new InputStreamReader(Thread.currentThread().getContextClassLoader.getResourceAsStream(propertiesName), "UTF-8"))
    prop
  }
}

File: gmall-canal/src/main/java/com/atguigu/utils/PropertiesUtil.java
package com.atguigu.utils;

import java.io.IOException;
import java.util.Properties;

/**
 * @author Chijago
 * @create 2020-02-21 20:58
 */
public class PropertiesUtil {
    public static Properties load(String path) {
        Properties properties = new Properties();

        try {
            properties.load(ClassLoader.getSystemResourceAsStream(path));
        } catch (IOException e) {
            e.printStackTrace();
        }
        return properties;
    }
}
